### DISCUSSION TITLE
could we have color-0 as an alias of white please?

### DISCUSSION THREAD

**Discussion by jayenne** [tailwindlabs/tailwindcss#4829] [Ideas]
hi,
I'm using colors as a @props for components and find a good argument for having color-0 being an alias for white - expecially when using dark mode or themes.
consider this example of a common component:
```
@props(['text' => 'gray', 'bg' => 'gray', 'border' => 'gray'])
<a {{ $attributes->merge(['class' => 'hover:text-'.$text.'-500 active:bg-'.$bg.'-900 focus:outline-none focus:border-'.$bg.'-900 text-'.$text.'-800 hover:text-'.$text.'-0 dark:text-'.$text.'-0 dark:hover:text-'.$text.'-800 bg-'.$color.'-0 hover:bg-'.$bg.'-800 dark:bg-'.$bg.'-800 dark:hover:bg-'.$bg.'-0 ]) }} >
    {{ $slot }}
</a>
````
all *-gray-0 would be == *-white

**Comment by Martleby** [comment]
Hi @jayenne 

A word of caution: I think you will run into trouble using these dynamic class names when you're running in production. When you run purge or the newer JIT to shrink your CSS down these classes won't be found and won't end up in your final CSS - meaning your design won't look how you expect/intend.

In you dev environment - if you can - I'd recommend using the new JIT feature of Tailwind. It's a very cool feature that means your dev environment CSS matches your production CSS.

All the best,
Martin.

**Reply by jayenne** [reply]
that sounds a perfectly acceptable answer. I hadn't thought of that sitution ...and use JIT's compiler. thank you for reminding me of this.
